src/analysis/functions.rs

```diff
@@ -514,6 +514,11 @@ fn analyze_function(
         .filter_map(|f| f.version)
         .min()
         .or(func.version);
+    let name = configured_functions
+        .iter()
+        .filter_map(|f| f.rename.clone())
```

Comment:
Shouldn't you filter here first to make sure `f.name == name`? Or does that happen elsewhere already?

Reply:
Indeed!

Reply:
Does the `filter_map` just above also miss that? Maybe the filtering is actually happening somewhere else already and that's why it works at all? Check the code in more detail please :)

Reply:
I can confirm that it's not good. Fixing it. :)

Reply:
configured_functions is filtered by function name by caller,
only problem is it pattern that applied to many functions,
and IMHO its not problem.
I more worry about docs.

Reply:
That's a good point.

Reply:
I'll need to check for the doc generation. For now, it's very certainly broken in this case.